sdk/monitor/azure-monitor-query/samples/sample_large_query.ipynb

```diff
@@ -0,0 +1,630 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Query with large result sets\n",
+    "\n",
+    "This sample notebook demonstrates how to query large amounts of data using the Azure Monitor Query client library.\n",
+    "\n",
+    "Due to Log Analytics [service limits](https://learn.microsoft.com/en-us/azure/azure-monitor/service-limits#la-query-api), sometimes it may not be possible to retrieve all the expected data in a single query. For example, the number of rows returned or the maximum size of the data returned may exceed the stated limits. One approach for overcoming these limits is to split the queries into multiple smaller queries using different time ranges.\n",
```

Comment:
```suggestion
    "Due to Log Analytics [service limits](https://learn.microsoft.com/azure/azure-monitor/service-limits#la-query-api), sometimes it may not be possible to retrieve all the expected data in a single query. For example, the number of rows returned or the maximum size of the data returned may exceed the stated limits. One approach for overcoming these limits is to split the queries into multiple smaller queries using different time ranges.\n",
```